### ISSUE TITLE
Single events slideshow being overlapped on some events

### ISSUE THREAD

**Issue by brashrebel** [realbig/RBM_TH_EPIC#32]
I clicked through a few events on epicraces.com and half of them show the bottom sliver of the slideshow with the title box covering most of it. The others were fine.

**Comment by AnaHotaling** [comment]
This happens to me a LOT. I just reload the page and it's good.

**Comment by d4mation** [comment]
I was able to reproduce the bug viewing [this Event](http://epicraces.com/event/epic-kids-splash-and-dash/) and it seems to be related to the Image Load Times. That page took especially long to load the slideshow in comparison to every other one that I tried.

When I opened the Inspector to figure out what was going on, that forced the DOM to resize and that made the JavaScript realize that the images were properly loaded now and it could stop only showing 19px of the height.

Now that it is cached, I'm not having the problem with that one any more at all, which leads me to think it is simply an issue with JavaScript guessing the height of the element incorrectly.

I'll try throwing a `min-height` of 300px on that element and its children (The height it seems to always be anyway), which should hopefully help alleviate the problem.
